notification-template/blade/DataSource/bin/start-jar.sh

```diff
@@ -28,6 +28,7 @@ if [ $confreading = 1 ]; then
    java -jar "$jar" "$@"
    exit $?
 else
-   java -cp "$classpath" -jar "$jar" "$@" 2> "$LOGDIR"/java-$BLADENAME.log >/dev/null &
+   start_command="java -cp "$classpath" -jar "$GC_OPTIONS" "$jar" "$@" 2> "$LOGDIR"/java-$BLADENAME.log >/dev/null &"
```

Comment:
we can run without eval, just like $start_command, but we can just run like previous. for it will just specify one jvm option, and recognize rest of them as the setting of the option.